ExomeDepth/run_ExomeDepth.py

```diff
@@ -36,14 +36,14 @@ def get_gender(bam):
     else:
         bam_base = os.path.basename(bam)
         if re.search('[C|P]M', bam_base):
-            print("Sample {0} has a unknown gender based on chrX reads, but resolved as male based on sampleID".format(
-                bam_base)
-            )
+            print(
+                "Sample {0} has a unknown gender based on chrX reads, but resolved as male based on sampleID"
+            ).format(bam_base)
             return "male"
         elif re.search('[C|P]F', os.path.basename(bam)):
-            print("Sample {0} has a unknown gender based on chrX reads, but resolved as female based on sampleID".format(
-                bam_base)
-            )
+            print(
+                "Sample {0} has a unknown gender based on chrX reads, but resolved as female based on sampleID"
+            ).format(bam_base)
```

Comment:
`.format()` links to `print()`?

```python
print(
    "Sample {0} has a unknown gender based on chrX reads, but resolved as female based on sampleID".format(bam_base)
)
```

Reply:
Did you test this?

Reply:
Line-length made it newline, but forgot the additional ()